src/components/BuildStatus/__tests__/BuildStatus.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render } from '@testing-library/react';
+import { BuildStatusType } from '../../../scripts/Build';
+import { BrowserRouter } from 'react-router-dom';
+import { BuildStatus } from '../BuildStatus';
+
+test('renders BuildStatus', () => {
+  render(
+    <BrowserRouter basename="/pnc-web">
+      <BuildStatus
+        identifier="20180911-1037"
+        status={BuildStatusType.CANCELLED}
+        user="jvanko"
+        date={new Date(2021, 11, 3, 10, 24, 43, 13)}
+      />
+    </BrowserRouter>
+  );
```

Comment:
Also, try to examine whether the data loaded was shown properly by using the `expect` assertion, see https://github.com/DnsZhou/pnc-web-ui-react/blob/NCL-6755/src/components/ProjectsList/__tests__/ProjectsList.test.tsx#L25-L30

You can find how to use the assertion from the document of React Testing Library here: https://testing-library.com/docs/react-testing-library/example-intro#assert

remember to add another test instead of putting it in `renders BuildStatus` test. You can also wrap all these two tests into a `describe` statement(see https://github.com/DnsZhou/pnc-web-ui-react/blob/NCL-6755/src/components/ProjectsList/__tests__/ProjectsList.test.tsx#L6)

Reply:
Also, try to change all <BrowserRouter> into <MemoryRouter>, since this is mostly used for testing because it won't change the URL in your browser instead it keeps the URL changes in memory.
see https://v5.reactrouter.com/web/api/MemoryRouter for introduction about MemoryRouter and BrowserRouter.